R/utils-get_hub.R

```diff
@@ -0,0 +1,63 @@
+#' Get hub configuration fields
+#'
+#' @inheritParams read_config
+#' @inheritParams get_round_idx
+#'
+#' @return * `get_hub_timezone`: The timezone of the hub
```

Comment:
I was wondering what the star was for and then I looked at the docs and realized its to make the arguments a list. This is clever!